### DISCUSSION TITLE
下書き本文をブラウザ検索できない

### DISCUSSION THREAD

**Discussion by NurseAngel** [increments/qiita-discussions#581] [要望・アイデア・不具合]
だいたい記事に書いたとおりですが、
ある程度以上の長い記事を書くと、ブラウザ検索で下書き入力画面のテキストが引っかからなくなります。
https://qiita.com/rana_kualu/items/9844d92766fad7f47e2f

通常の何もしていないinputやtextareaでは1000行程度ではこんなことにならないので、何かスクリプトが悪さをしていると思いますが、そこまでは調べていません。

\# あと下書き画面がCtrl+Fを奪い取るのはやめてほしい

**Comment by tanaka-asami415** [comment]
@NurseAngel 
ご不便をおかけして、申し訳ありません。
パフォーマンスの仕様上ブラウザの検索機能を制限しております。

エディタ上で検索機能が使えるようになっていますので（Ctrl+Fがショートカットとなっています）
こちらをご使用いただけると幸いです🙇

![スクリーンショット 2023-12-11 13 12 00](https://github.com/increments/qiita-discussions/assets/103916185/f0b1c726-0fef-4b60-bfde-2e747b9352c0)
